src/General/Alert/Alert.tsx

```diff
@@ -9,58 +9,43 @@ import {
   CloseIcon,
 } from '../Icon/components';
 import {
+  AlertClose,
   AlertContainer,
-  AlertContent,
-  AlertMessage,
   AlertIcon,
-  AlertTypeIconContainer,
+  AlertMessage,
 } from './AlertStyle';
 
 import { Greyscale, PrimaryColor, SecondaryColor } from '../../Utils/Colors';
+import { isUndefined } from 'lodash';
+import { DelayedUnmount } from '../../Utils/DelayedUnmount';
 
 export const Alert = ({
-  isOpen,
+  isOpen: externalIsVisible = true,
```

Comment:
This will break `const isVisible = isUndefined(externalIsVisible) ? internalIsVisible : externalIsVisible;`.

Reply:
Right, should not have committed that!